compositor_pipeline/src/queue/internal_queue.rs

```diff
@@ -131,4 +134,22 @@ impl InternalQueue {
     pub fn get_next_output_buffer_pts(&self) -> Duration {
         Duration::from_secs_f64(self.sent_batches_counter as f64 / self.output_framerate.0 as f64)
     }
+
+    pub fn register_input_listener(
+        &mut self,
+        input_id: InputId,
+        callback: Box<dyn FnOnce() + Send>,
+    ) {
+        self.inputs_listeners
+            .entry(input_id)
+            .or_insert_with(std::vec::Vec::new)
```

Comment:
Is `std::vec::` part needed here?

Reply:
I added `vec![]` and clippy probably changed it